source/openapi-admin-v3.yaml

```diff
@@ -2596,9 +2599,51 @@ paths:
                 type: array
                 items:
                   "$ref": "#/components/schemas/Deployment"
+  "/groups/{groupId}/apps/{appId}/deployments/{deploymentId}":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/DeploymentId"
+    get:
+      tags:
+        - deploy
+      operationId: adminGetDeployment
+      summary: Get a deployment
+      description: Get information about a recent deployment of the application.
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/Deployment"
+  "/groups/{groupId}/apps/{appId}/deployments/{deploymentId}/redeploy":
     parameters:
       - "$ref": "#/components/parameters/GroupId"
       - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/DeploymentId"
+    get:
+      tags:
+        - deploy
+      operationId: adminRedeployDeployment
+      summary: Redeploy a deployment
+      description: Redeploy a previously deployed deployment.
```

Comment:
Too many deploys. Could we reword this slightly? Maybe something like:

"Redeploy a previously-deployed version of the App" or something?